test/gas/gasLimit.js

```diff
@@ -0,0 +1,16 @@
+const assert = require("assert");
+const to = require("../../lib/utils/to.js");
+const preloadWeb3 = require("../helpers/web3/preloadWeb3");
+
+describe("options:gasLimit", () => {
+  const mnemonic = "candy maple cake sugar pudding cream honey rich smooth crumble sweet treat";
```

Comment:
Probably not necessary for this review, but we should move away from mnemonics being specified in tests.

Reply:
That is, unless a test requires a very specific mnemonic (instead of a mnemonic that doesn't change from run to run) the mnemonic in question should be defined in one of the test helpers - `getWeb3` seems like a good choice from what I've seen of it so far

Reply:
Another options would be to store mnemonics in a .env implementation.

Reply:
The issue I'm raising here isn't that mnemonics should be kept secret, but rather that there's no need for tests which need an unchanging mnemonic to define their own. It's about encapsulation and code duplication rather than security. These are test phrases published in the open - we should expect that every one of them is being scraped.

Reply:
further, adding a requirement for a .env file that isn't checked into source control would increase the burden of running tests, meaning less people would be likely to run tests before submitting PRs, and fewer still would write tests as part of their contributions

Reply:
So far it makes sense to move the mnemonics into the `getWeb3`

Reply:
@davidmurdoch Perhaps using a seed to generate mnemonics/accounts may be more secure and traceable.

Reply:
@ccowell what's the security concern here?

Reply:
No huge security concerns other than having mnemonics out there. More about best practices.

Reply:
@benjamincburns 	I think the concern is that people may use our tests as examples of how to use ganache-cli/core, and we're committing our mnemonics. Obviously everyone _should_ already know that you shouldn't be doing this with mainnet mnemonics, but these sorts of security blunders still happen.

With that said, a seed is basically still a mnemonic (it's actually just used to generate one, internally), but at least wallets don't provide users with a `seed` they can copy and then paste into their code.

So... to reiterate: the concern is that someone may say to themselves "Ganache commits mnemonics to their repo, I have a mnemonic, I have a repo, I should commit it to my repo!". By using a seed in our tests instead of a mnemonic we may prevent users from making this mistake.

Alternatively, we could just leave big old `"/* KEEP YOUR MAINNET MNEMONICS SECURE AND PRIVATE OR YOU WILL LOOSE ALL YOUR FUNDS */"` comments wherever we do use a mnemonic, but a `seed` still seems easier.

Reply:
I'm leaning towards seeding atm.

Reply:
Replaced mnemonic with a randomly generated seed for refactored tests only.